Version-1.8-CHANGELOG (#428)
KxSystems/kx-vscode: CHANGELOG.md

All notable changes to the **kdb VS Code extension** are documented in this file.

# v1.8.0

### Enhancements

- Show KDB+ process explorer item content when clicked
- Add ability to export and import connections

> - All files in workspace are considered when using language server features

Reviewer: ```suggestion
- All files in the workspace are considered when using language server features
```

- Show call hierarchy is implemented in language server
- Query history shows an ellipsis of the query execution text
- Add limit option to datasource

### Fixes

> - Fixed KDB results columns resizing back to defaults everytime a datasource is run

Reviewer: ```suggestion
- Fixed KDB results columns resizing back to default sizes every time a datasource was run
```
